repos/fdbt-site/tests/pages/ticketConfirmation.test.tsx

```diff
@@ -498,7 +499,7 @@ describe('pages', () => {
                 });
                 const confirmationElements = buildSchoolTicketConfirmationElements(ctx);
                 expect(confirmationElements).toContainEqual(confirmationElementStructure);
-                expect(confirmationElements).toHaveLength(2);
+                expect(confirmationElements).toHaveLength(13);
```

Comment:
I assume this number has increased by 11 because its generating confirmation elements for a point to point period. This test needs renaming to show that, and we will need a new test for a non-period point to point academic ticket.

Reply:
The number is incremented as the function calling is changed.